app/views/ResultsPageView.scala.html

```diff
@@ -33,7 +33,7 @@
 
 @layout(pageTitle = titleNoForm(messages("resultsPage.title"))) {
 
-    @displayBanner(calculatorResult)
+    @addBannerScreenReader(displayBanner(calculatorResult), calculatorResult)
```

Comment:
Why dont we incorporate this functionality into the displayBanner method? Looks like every banner needs a bit of text for screen reader